File: Back/src/JitResidencial.Domain/Produto.cs
using System;
using System.Collections.Generic;

namespace JitResidencial.Domain
{
    public class Produto

    {
        public int Id { get; set; }
        public string CodigoBarras { get; set; }
        public string NomeProduto { get; set; }
        public int Quantidade { get; set; } 
        public string Volume { get; set; }
        public string DataValidade { get; set; }
        public string DataInclusao { get; set; }
        public string DataAlteracao { get; set; }
        public int UnidadeMedidaId {get; set;}
        public int CategoriaId  {get; set;}
        public int MovimentoId { get; set; }
        public int EstoqueId { get; set; }
        public int ListaPrecoId { get; set; }
        public int FornecedorId { get; set; }        
    }
}

File: Front/JitResidencial-App/src/app/models/Produto.ts

export interface Produto {

 id: number;
 codigoBarras:  string;
 nomeProduto:  string;
 quantidade:  number;
 volume:  string;
 dataValidade:  string;
 dataInclusao?:  Date;
 dataAlteracao?:  Date;
 unidadeMedidaId: number;
 categoriaId: number;
 movimentoId: number;
 estoqueId: number;
 listaPrecoId:  number;
 fornecedorId: number;
}